api/main.go

```diff
@@ -0,0 +1,112 @@
+// This is the main package for the first-api application.
+package main
```

Comment:
Folder structuring 
1. cmd
    - start your server, small main fn, call internal/pkg
2. internal
    - code which you do not want to expose to the rest of the world. 
3. pkg
   - reusable code outside your application also